src/adservice/src/main/java/hipstershop/AdService.java

```diff
@@ -106,30 +93,32 @@ private void stop() {
     @Override
     public void getAds(AdRequest req, StreamObserver<AdResponse> responseObserver) {
       AdService service = AdService.getInstance();
-      Span span = tracer.getCurrentSpan();
+      // Span span = tracer.getCurrentSpan();
       try {
-        span.putAttribute("method", AttributeValue.stringAttributeValue("getAds"));
+        // span.putAttribute("method", AttributeValue.stringAttributeValue("getAds"));
```

Comment:
How hard is it to have the attribute in signalfx instrumentation? Could you a TODO here?

Reply:
Added comment